### PR TITLE
Callout: Add window segments

### PR DESCRIPTION
**Pull request by jspurlin**
#### Pull request checklist
- [X] Include a change request file using `$ yarn change`

#### Description of changes
This PR makes callouts aware of the WindowSegments both vertically and horizontally

#### Focus areas to test
Verified that the callouts respond appropriately to the presence of WindowSegments

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10368)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Sorry for the obvious question: But what are "WindowSegments" and why are we concerned about them? Is there a related issue here?

File: packages/utilities/src/dom/getRenderedContainer.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { IPoint } from '../IPoint';
+import { Rectangle } from '../Rectangle';
+
+export function getRenderedContainer(
```
Comment (jdhuntington) [inline packages/utilities/src/dom/getRenderedContainer.ts]: Is there another name that would fit for this? Is `getContainerSurroundingContent` accurate? Without the context of `callout`, it's hard to discern how this should be used.
Alternately, if this is only going to be used by callout, could this just be a private method in the scope of callout?

Comment (jspurlin) [reply packages/utilities/src/dom/getRenderedContainer.ts]: I wanted to keep `Rendered` in the name to make it clear that this refers to the rendered container, not the logical container.

Other things like dialog (and I think coachmark) will be using this function

**Comment by joschect** [comment]
What exactly are windowSegments? I can't find documentation on them. Are they new? And where does getBoundingWindowRects come from?

**Pull request closed without merging**